### PR TITLE
Styling things

### PR DESCRIPTION
**Pull request by zach-bergman**
- stopped images auto scrolling
- sped up image switching when clicking carousel arrows
- cleaned up extra link on bottom of cards
- on search results page, when rc button is clicked, the page auto scrolls down to see the form
- changed favorite icon on show page to be an outlined heart, when clicked, the heart fills in and scales up. also added a data tag on favorite icon html that is switched from false to true when icon is clicked

### CODE REVIEW

**Review by RodrigoACG — APPROVED** [review]
This looks amazing and will definitely speed up the process!!!

File: app/views/listings/show.html.erb [hunk 1]
```diff
@@ -35,8 +35,8 @@
                   <span><i class="fas fa-expand-arrows-alt"></i> <%= @listing.sqft %> m² </span>
                 </div>
               </div>
-              <a href="#" class="position-absolute" style="top: 10px; right: 10px;">
-                <i class="fas fa-heart" style="font-size: 24px; color: red;"></i>
+              <a href="#" class="position-absolute favorite-icon" data-favorite="false" style="top: 10px; right: 10px;">
+                <i class="far fa-heart" style="font-size: 28px; color: red;"></i>
```
Comment (jdmchugh111) [inline app/views/listings/show.html.erb:39]: great work reconfiguring the favorite icon

File: app/views/search/index.html.erb [hunk 2]
```diff
@@ -49,20 +47,21 @@
...
+  <!-- Collapsible Form -->
+  <div class="collapse" id="incomeFormContainer" style="margin-top: 20px;">
+    <div class="card card-body">
+      <%= form_with url: reality_check_index_path, method: :get, local: true do |form| %>
+        <div class="form-group">
+          <%= form.label :monthly, "Enter your monthly income:" %>
+          <%= form.number_field :monthly, class: "form-control", required: true, min: 0 %>
+        </div>
+        <%= form.submit "Submit", class: "btn btn-primary" %>
+      <% end %>
```
Comment (jdmchugh111) [inline app/views/search/index.html.erb:64]: dude you're the frontend goat!

**Review by jdmchugh111 — APPROVED** [review]
you're really going above and beyond - thanks so much for putting in the extra coding time this weekend

**Pull request merged**